tests/test_commands.py

```diff
@@ -50,6 +51,9 @@ def test_create_item(self):
             self.assertEqual(raster_band.data_type, DataType.UINT8)
             self.assertEqual(raster_band.unit, "none")
 
+        sci_ext = ItemScientificExtension.ext(item)
+        self.assertEqual(sci_ext.doi, "10.5066/F7QN651G")
+
```

Comment:
This is a good test, but IMO it belongs in `tests/test_stac.py` -- we don't need to go through the CLI to test that the scientific extension is added correctly.

Reply:
what about the raster extension? it looks like the tests for this were also added in `test_commands.py` and not `test_stac.py` in one of the earlier PR's...